README.md

```diff
@@ -13,9 +13,9 @@ Google Cloud Java libraries. For manually generating the dashboard, see
 
 # Java Library Best Practices
 
-[The Java Library Best Practices](./docs) are a set of rules
-that we believe will minimize problems for consumers of interconnected Java
-libraries.
+[The Java Library Best Practices](https://googlecloudplatform.github.io/cloud-opensource-java/)
```

Comment:
https://googlecloudplatform.github.io/cloud-opensource-java returns 404. Will it be available?

Reply:
That's what this PR does